### PR TITLE
Press page

### PR DESCRIPTION
**Pull request by taggartbg**
See #142.

Screen Shot 2019-10-17 at 11 16 11 AM
Screen Shot 2019-10-17 at 11 21 36 AM
Screen Shot 2019-10-17 at 11 19 54 AM
Screen Shot 2019-10-17 at 11 20 03 AM

### CODE REVIEW

**Comment by Shadowfiend** [comment]
Quick note: looks like we're missing the footer on the press page.

File: src/PressContent.jsx [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+                                <div className="media-kit-link">
+                                    <Newspaper />
+                                    <div className="media-kit-link-text">
+                                        Media Kit
+                                    </div>
+                                    <div className="media-kit-link-button">
+                                        download
+                                    </div>
+                                </div>
+                            </a>
```
Comment (Shadowfiend) [inline src/PressContent.jsx]: It feels weird to have a download button and not have it be the only hit area. Any reason why the button wouldn't be the only link?

In a related vein, this feels like a pretty heavy way to do the “label with icon and download button” pattern. Not super worried about that bit for the PR though.

Comment (taggartbg) [reply src/PressContent.jsx]: I"m happy to change it, but FWIW here are my justifications:

- I figured more is better wrt actionable screen area, nothing is particularly gained by making the label not actionable.
- I tend prefer verbose / easy-to-edit over concise and creative, but I'm happy to make this a bit more semantic if you want to see real button and label elements.

Comment (Shadowfiend) [reply src/PressContent.jsx]: I could see the actionable screen area argument, yeah. Threw VoiceOver at it and it looks like it reads properly, so no big deal.

File: src/PressContent.jsx [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+                                key={entry.url}
+                                />
+                        ))}
+                    </div>
+                </PageSection>
+                {
+                    (allPressEntries.length > 10) && (this.state.pressEntries.length === 10)
+                    ? (
+                        <PageSection id="see-all">
+                            <div className="see-all-button" onClick={this.handleShowAll}>
```
Comment (Shadowfiend) [inline src/PressContent.jsx]: Would be good to make this a `button` to reflect its actionable nature and give it a good tab index.

**Comment by Shadowfiend** [comment]
If you feel you can clear the draft status here, I think https://github.com/keep-network/website/pull/146#discussion_r336633063 is the only thing I see blocking a merge + deploy.

**Comment by taggartbg** [comment]
@Shadowfiend Final updates pushed after discussion w/ Liz

Screen Shot 2019-10-22 at 2 48 46 PM

**Pull request merged**